cmd/jujud/reboot/reboot.go

```diff
@@ -164,3 +165,58 @@ func (r *Reboot) waitForContainersOrTimeout() error {
 		return errors.Trace(err)
 	}
 }
+
+func writeScript(args []string, after int) (string, error) {
+	tpl := `#!/bin/bash
```

Comment:
We're setting the shebang line here to bash but we're executing it with bourne shell below. Should be a little more explicit.